xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -714,6 +714,20 @@ bool CPVRChannelGroup::AddToGroup(const std::shared_ptr<CPVRChannel>& channel, c
   return bReturn;
 }
 
+bool CPVRChannelGroup::AppendToGroup(const std::shared_ptr<CPVRChannel>& channel)
+{
+  CSingleLock lock(m_critSection);
+
+  unsigned int channelNumberMax = 0;
+  for (const auto member : GetMembers())
```

Comment:
`GetMembers()` creates a copy of `m_sortedMembers`.

I would go for `for (const auto& memberPair : m_members)` to avoid the copy. Or, if you actually need `m_sortedMembers`, access it directly.

Reply:
My bad, should have seen that.